### PR TITLE
21256 - Add packet loss % to stats

### PR DESCRIPTION
**Pull request by Triplelexx**
**Test Plan:**

In the existing stats, there is a column with 'Audio ping:' , etc which gives the milliseconds of ping to the audio server. 
This PR should modify that line to now show 'Audio ping/loss:' with the loss shown as a percentage.

To test: You can use some software to simulate dropping packets of outbound network data to cause the loss.
Using http://jagt.github.io/clumsy/ you would enable drop. The chance percentage value should be reflected in the stats.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: f588612fc30b9d457668456aa622a65394e17a42 available at: https://gist.github.com/359a77ad72455dfb8b18864c5bdd451e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-f588612.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-f588612.dmg

File: interface/src/ui/Stats.cpp [hunk 1]
```diff
@@ -164,7 +164,9 @@ void Stats::updateStats(bool force) {
     SharedNodePointer avatarMixerNode = nodeList->soloNodeOfType(NodeType::AvatarMixer);
     SharedNodePointer assetServerNode = nodeList->soloNodeOfType(NodeType::AssetServer);
     SharedNodePointer messageMixerNode = nodeList->soloNodeOfType(NodeType::MessagesMixer);
+    auto mixerStream = _audioStats->data()->getMixerStream();
     STAT_UPDATE(audioPing, audioMixerNode ? audioMixerNode->getPingMs() : -1);
+    STAT_UPDATE(audioPacketLoss, mixerStream ? roundf(mixerStream->lossRateWindow() * 100.0f) : -1);
```
Comment (kencooke) [inline interface/src/ui/Stats.cpp]: should probably be (int)roundf(...)

Comment (Triplelexx) [inline interface/src/ui/Stats.cpp]: that's what I thought, other integers are used without C cast as well.

Comment (Triplelexx) [inline interface/src/ui/Stats.cpp]: STAT_UPDATE appears to convert to auto.

Comment (kencooke) [inline interface/src/ui/Stats.cpp]: The auto ends up ```float```, but audioPacketLoss is ```int``` and results in several implicit conversions. Not sure why this doesn't throw a warning and fail the build.

Comment (kencooke) [inline interface/src/ui/Stats.cpp]: Sorry, I didn't notice the existing code was already doing implicit casting. Thanks for cleaning up. We compile with "warning on implicit conversions" and "treat warnings as errors" so this usually fails to compile.

**Comment by hifi-gustavo** [comment]
Build console output for commit: fcfd206e9c52601506c3ab3fe73657c18271e07a available at: https://gist.github.com/359a77ad72455dfb8b18864c5bdd451e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-fcfd206.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-fcfd206.dmg

**Comment by PhilipRosedale** [comment]
Hmmm.... trying to test but I'm not getting clumsy to give me packet loss.  Will try on mac.

**Comment by skytoad** [comment]
@PhilipRosedale I'm experiencing the same issue, though I dont have a mac to test with

**Comment by PhilipRosedale** [comment]
QA:  I believe you are counting upstream packet loss here, yes?  I think it would be better to display the larger of upstream and downstream loss.... can we do that?

**Comment by Triplelexx** [comment]
I think this should do the job. I have just wasted over an hour trying to get Clumsy working again, or to use an alternative.

**Comment by hifi-gustavo** [comment]
Build console output for commit: ec6885a2fbf04f6fa01e2bbae3b54bc6f7c738ec available at: https://gist.github.com/359a77ad72455dfb8b18864c5bdd451e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-ec6885a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-ec6885a.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 54eb7c10c6f59ba794a792a2e43f9a61a1894c31 available at: https://gist.github.com/359a77ad72455dfb8b18864c5bdd451e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-54eb7c1.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-54eb7c1.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7936637e9c43d8f6b2b89bd4608fa1a75b58552e available at: https://gist.github.com/359a77ad72455dfb8b18864c5bdd451e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-7936637.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10156-7936637.dmg

**Comment by Triplelexx** [comment]
Main problem I was having was Clumsy displaying error code 3.

Failed to start with error code 3.
Run cmd and use these two lines:

            sc stop WinDivert1.0
            sc delete WinDivert1.0 
            
Is the instructions, it's actually using version 1.1 though and WinDivert latest is 1.2

**Comment by PhilipRosedale** [comment]
Tested and works great for me.  Tested on windows and mac.

**Pull request merged**